Touch Bar Simulator/TouchBarWindow.swift

```diff
@@ -295,6 +300,17 @@ final class TouchBarWindow: NSPanel {
 	func removeTitlebar() {
 		styleMask.remove(.titled)
 	}
+	
+	func roundCorners(_ corners: CACornerMask) {
+		self.contentView?.layer?.cornerRadius = 7.5
+		self.contentView?.layer?.maskedCorners = []
+
+		self.contentView?.layer?.maskedCorners = corners
+		self.contentView?.layer?.borderWidth = 0.0
+		self.contentView?.layer?.borderColor = .clear
+		self.isOpaque = false
+		self.backgroundColor = .clear
```

Comment:
Shouldn't this be set to the opposite here https://github.com/sindresorhus/touch-bar-simulator/pull/54/files#diff-e6aaf782da40568071255e5e366b109dR284?